backend/api/Services/IsarService.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace Api.Services
+{
+    internal interface IsarServiceInterface
```

Comment:
```suggestion
    internal interface IIsarService
```
Naming convention for interfaces

Reply:
Fixed